index.bzl

```diff
@@ -63,21 +63,6 @@ COMMON_REPLACEMENTS = {
     "\"@bazel/([a-zA-Z_-]+)\":\\s+\"(file|bazel)[^\"]+\"": "\"@bazel/$1\": \"0.0.0-PLACEHOLDER\"",
 }
 
-def node_modules_filegroup(packages, patterns = [], **kwargs):
```

Comment:
looks like also good to document in BREAKING CHANGE

Reply:
+1